### DISCUSSION TITLE
RESTEASY004687 - Please close clients yourself

### DISCUSSION THREAD

**Discussion by toas-jyskebank-dk** [quarkusio/quarkus#38019] [Community]
I get a lot of these entries in my application log:

`WARN [org.jbo.res.cli.jax.i18n] (Cleaner-0) RESTEASY004687: Closing a class org.jboss.resteasy.client.jaxrs.engines.ManualClosingApacheHttpClient43Engine$CleanupAction instance for you. Please close clients yourself`.

My application is running Quarkus version 3.3.2 and it is using a number of microprofile REST clients defined like this:

```
@ApplicationScoped
@RegisterRestClient(configKey = "dataproxy")
@RegisterProvider(ClientLogger.class)
public interface DataproxyClient extends org.openapi.generated.dataproxy.api.ProduktApi
```

Do I need to worry about that warning?... if yes, how should I fix it?

**Comment by eldercomputacao** [comment]
Try extending AutoCloseable, I see: https://stackoverflow.com/questions/64137112/quarkus-restclient-close
